Makefile

```diff
@@ -112,10 +112,12 @@ build_tools:
 	$(HOSTGO) build -o ./tools/custom_builder/custom_builder$(EXEEXT) ./tools/custom_builder
 	$(HOSTGO) build -o ./tools/license_checker/license_checker$(EXEEXT) ./tools/license_checker
 	$(HOSTGO) build -o ./tools/readme_config_includer/generator$(EXEEXT) ./tools/readme_config_includer/generator.go
+	$(HOSTGO) build -o ./tools/config_includer/generator$(EXEEXT) ./tools/config_includer/generator.go
 	$(HOSTGO) build -o ./tools/readme_linter/readme_linter$(EXEEXT) ./tools/readme_linter
 
 embed_readme_%:
-	go generate -run="readme_config_includer/generator$$" ./plugins/$*/...
+	go generate -run="tools/config_includer/generator\s?.*$$" ./plugins/$*/...
```

Comment:
Why the changed call to generator? Is that a regex that got out? 

This doesn't appear to work on windows:

```s
go generate -run="tools/config_includer/generator\s?.*$" ./plugins/inputs/...
2023/05/31 10:26:35 Reading template "\\plugins\\inputs\\http\\sample.conf.in" failed: open \plugins\inputs\http\sample.conf.in: The system cannot find the path specified.
plugins\inputs\http\http.go:1: running "../../../tools/config_includer/generator": exit status 1
make: *** [Makefile:119: embed_readme_inputs] Error 1
```

Reply:
Uhhh. I need to fix the regexp then. I changed this to allow arguments passed to the generator e.g. if you want multiple templates to be rendered etc.

Reply:
Should be fixed now.